src/components/Explorer/queryTemplate.js

```diff
@@ -71,71 +130,84 @@ ${isTi7Team ? 'AND teams.team_id IN (5, 15, 39, 46, 2163, 350190, 1375614, 18383
-${maxPatch ? `AND match_patch.patch <= '${maxPatch.value}'` : ''}
-${hero ? `AND player_matches.hero_id = ${hero.value}` : ''}
-${player ? `AND player_matches.account_id = ${player.value}` : ''}
-${league ? `AND matches.leagueid = ${league.value}` : ''}
-${playerPurchased ? `AND (player_matches.purchase->>'${playerPurchased.value}')::int > 0` : ''}
-${minDuration ? `AND matches.duration >= ${minDuration.value}` : ''}
-${maxDuration ? `AND matches.duration <= ${maxDuration.value}` : ''}
-${side ? `AND (player_matches.player_slot < 128) = ${side.value}` : ''}
-${result ? `AND ((player_matches.player_slot < 128) = matches.radiant_win) = ${result.value}` : ''}
-${team ? `AND notable_players.team_id = ${team.value}` : ''}
-${organization ? `AND team_match.team_id = ${organization.value} AND (player_matches.player_slot < 128) = team_match.radiant` : ''}
-${laneRole ? `AND player_matches.lane_role = ${laneRole.value}` : ''}
-${region ? `AND matches.cluster IN (${region.value.join(',')})` : ''}
-${minDate ? `AND matches.start_time >= extract(epoch from timestamp '${new Date(minDate.value).toISOString()}')` : ''}
-${maxDate ? `AND matches.start_time <= extract(epoch from timestamp '${new Date(maxDate.value).toISOString()}')` : ''}
-${tier ? `AND leagues.tier = '${tier.value}'` : ''}
-${isTi7Team ? 'AND teams.team_id IN (5, 15, 39, 46, 2163, 350190, 1375614, 1838315, 1883502, 2108395, 2512249, 2581813, 2586976, 2640025, 2672298, 1333179, 3331948, 1846548)' : ''}
-${group ? `GROUP BY ${groupVal}` : ''}
-${group ? `HAVING count(distinct matches.match_id) >= ${having ? having.value : '1'}` : ''}
+${validate(selectArray) ? selectArray.map(x => `AND ${x.value} IS NOT NULL `).join('') : ''}
+${validate(minPatch) ? templ`match_patch.patch >= '${minPatch}'` : ''}
+${validate(maxPatch) ? templ`match_patch.patch <= '${maxPatch}'` : ''}
+${validate(hero) ? templ`player_matches.hero_id = ${hero}` : ''}
+${validate(player) ? templ`player_matches.account_id = ${player}` : ''}
+${validate(league) ? templ`matches.leagueid = ${league}` : ''}
+${validate(playerPurchased) ? templ`(player_matches.purchase->>'${playerPurchased}')::int > 0` : ''}
+${validate(minDuration) ? templ`matches.duration >= ${minDuration}` : ''}
+${validate(maxDuration) ? templ`matches.duration <= ${maxDuration}` : ''}
+${validate(side) ? templ`(player_matches.player_slot < 128) = ${side}` : ''}
+${validate(result) ? templ`((player_matches.player_slot < 128) = matches.radiant_win) = ${result}` : ''}
+${validate(team) ? templ`notable_players.team_id = ${team}` : ''}
+${validate(organization) ? templ`team_match.team_id = ${organization} AND (player_matches.player_slot < 128) = team_match.radiant` : ''}
+${validate(laneRole) ? templ`player_matches.lane_role = ${laneRole}` : ''}
+${validate(region) ? templ`matches.cluster IN (${region})` : ''}
+${validate(minDate) ? templ`matches.start_time >= extract(epoch from timestamp '${new Date(minDate).toISOString()}')` : ''}
```

Comment:
date filters don't seem to be getting applied to the query, e.g.:
https://deploy-preview-1406--opendota-stage.netlify.com/explorer?select=assists&&minDate=2018-01-24T08%3A00%3A00.000Z